docs/en/02_Developer_Guides/01_Templates/07_Caching.md

```diff
@@ -35,13 +35,11 @@ When we render `$Counter` to the template we would expect the value to increase
 
 ## Partial caching
 
-Partial caching is a feature that allows the caching of just a portion of a page. Instead of fetching the required data
-from the database to display, the contents of the area are fetched from a [cache backend](../performance/caching).
+Partial caching is a feature that allows caching of a portion of a page as a single string value. For more details read [its own documentation](partial_template_caching).
 
+Example:
 ```ss
-<% cached 'MyCachedContent', LastEdited %>
-    $Title
+<% cached 'MyCachedContent', $LastEdited if $IsCacheable %>
```

Comment:
I don't think the `if $IsCacheable` is a good example, because a)
- There's no function on DataObject or similar called `IsCacheable()`
- It's not common to check to see if a DataObject is cacheable, I don't think I've ever seen this done in an actual project.  Due to the rigid style of templates you know ahead of time what type of DataObject you're dealing with, and if you don't want to cache it then you just wouldn't add a `cached` block to the template